integration/containerd/confidential/agent_image.bats

```diff
@@ -111,11 +111,36 @@ setup() {
 	fi
 }
 
+@test "$test_tag Test unencrypted cosigned image with co-signature" {
```

Comment:
What if `unencrypted image signed with cosign`? `cosign` is a signing scheme name, in this way it may sound better. How do you like?